lib/fortnox/api/models/article.rb

```diff
@@ -0,0 +1,147 @@
+        # )
+        #
+        # filter(
+        #   :active
+        # )
+
+        #Url Direct URL to the record
+        attribute :url, Fortnox::API::Types::Nullable::String.with( read_only: true )
+
+        #Active If the article is active
+        attribute :active, Fortnox::API::Types::Nullable::Boolean
+
+        #ArticleNumber Article number. 50 characters
+        attribute :article_number, Fortnox::API::Types::Sized::String[ 50 ]
+
+        #Bulky If the article is bulky.
+        attribute :bulky, Fortnox::API::Types::Nullable::Boolean
+
+        #ConstructionAccount Account number for construction work (special VAT rules in Sweden).
+        # The number must be of an existing account.
+        attribute :construction_account, Types::Sized::Integer[ 0, 9_999 ]
+
+        #Depth The depth of the article in millimeters
+        attribute :depth, Types::Sized::Integer[ 0, 99_999_999 ]
+
+        #Description The description of the article
+        attribute :description, Fortnox::API::Types::Sized::String[ 200 ]
+
+        #DisposableQuantity Disposable quantity of the article.
+        attribute :disposable_quantity, Fortnox::API::Types::Nullable::Float.with( read_only: true )
+
+        #EAN EAN bar code
+        attribute :ean, Fortnox::API::Types::Sized::String[ 30 ]
+
+        #EUAccount Account number for the sales account to EU.
+        # The number must be of an existing account.
+        attribute :eu_account, Types::Sized::Integer[ 0, 99_99 ]
+
+        #EUVATAccount Account number for the sales account to EU with VAT.
+        # The number must be of an existing account.
+        attribute :eu_vat_account, Types::Sized::Integer[ 0, 99_99 ]
+
+        #ExportAccount Account number for the sales account outside EU
+        # The number must be of an existing account.
+        attribute :export_account, Types::Sized::Integer[ 0, 99_99 ]
+
+        #Height The height of the article in millimeters
+        attribute :height, Types::Sized::Integer[ 0, 99_999_999 ]
+
+        #Housework If the article is housework
+        attribute :housework, Fortnox::API::Types::Nullable::Boolean
+
+        #HouseWorkType The type of house work.
+        attribute :house_work_type, Types::HouseWorkType
+
+        #Manufacturer The manufacturer of the article
+        attribute :manufacturer, Fortnox::API::Types::Sized::String[ 50 ]
+
+        #ManufacturerArticleNumber The manufacturer's article number
+        attribute :manufacturer_article_number, Fortnox::API::Types::Sized::String[ 50 ]
+
+        #Note Text note
+        attribute :note, Fortnox::API::Types::Sized::String[ 10_000 ]
+
+        #PurchaseAccount Account number for purchase.
+        # The number must be of an existing account.
+        attribute :purchase_account, Types::Sized::Integer[ 0, 99_99 ]
+
+        #PurchasePrice Purchase price of the article
+        attribute :purchase_price, Fortnox::API::Types::Sized::Float[ 0.0, 99_999_999_999_999.9 ]
```

Comment:
@d-Pixie & @felixyz Accoding to the documentation, this attribute is a `float, 14 digits`. Should the decimal fraction be `.9` as here, or should it be `.0` as in `Fortnox::API::Types::DocumentRow`: `attribute :delivered_quantity, Types::Sized::Float[ 0.0, 9_999_999_999_999.0 ]`?

Reply:
I got this from the declaration of `InvoiceAdministrationFee` (float, 11 digits) in Customer:

```Ruby
attribute :invoice_administration_fee,
  Fortnox::API::Types::Sized::Float[ 0.0, 99_999_999_999.9 ]
```

(11 + 1)

But I really have no idea what the correct definition would be. Seems to me the number of _digits_ in a float can't be captured by a number range?